backend/server.js

```diff
@@ -1,35 +1,28 @@
-const express = require('express');
-const bodyParser = require('body-parser');
-const path = require('path');
-const MongoClient = require('mongodb').MongoClient
-require('dotenv').config()
+const express = require("express");
+const bodyParser = require("body-parser");
+const path = require("path");
+const MongoClient = require("mongodb").MongoClient;
+require("dotenv").config();
+
+const divisions = require("./Controllers/divisions.js");
 
 const app = express();
 const port = process.env.PORT || 5000;
 
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 
-// MongoDB Database
-MongoClient.connect(process.env.MONGODB_URI, {
-  useUnifiedTopology: true
-})
-  .then(client => {
-    const db = client.db('RGL')
-    const countersCollection = db.collection('site')
-    console.log('Connected to Database')
-  })
-
+app.get("/divisions", divisions);
 
-// Heroku deployment compatibility: 
-if (process.env.NODE_ENV === 'production') {
+// Heroku deployment compatibility:
+if (process.env.NODE_ENV === "production") {
```

Comment:
zmiana z ' na " jest celowa? Osobiście preferuje ', ale jak chcesz, to możemy używać ", byle by było spójnie

Reply:
Mam z tym problem niestety. Postaram się być bardziej spójny. Niech będzie zatem '.

Reply:
UPDATE: nasz wspólny formatter zmienia na ".